nestkernel/connection_manager.h

```diff
@@ -419,6 +423,10 @@ class ConnectionManager : public ManagerInterface
 
   //! Capacity growth factor to use beyond the limit
   double large_connector_growth_factor_;
+
+  //! Maximum distance between (double) spike times in STDP that is
+  //! still considered 0
```

Comment:
I think it would be useful to refer to the issue here.

Reply:
Yes, you are right. Done.